html/Turn of Tides Food Ingredient Additions

```diff
@@ -0,0 +1,89 @@
+food = {
+	
+	//DST Turn Of Tides Ingredient Additions	
+	kelp: {
+				name: 'Kelp Fronds',
+				uncookable: true,
+				isveggie: true,
+				health: -healing_tiny,
+				hunger: calories_tiny,
+				sanity: -sanity_small,
+				perish: perish_med,
+				stack: stack_size_smallitem,
+				dry: 'kelp_dried',
+				drytime: dry_superfast
+				mode: 'together'
+			},
+			
+	kelp_cooked: {
+				name: 'Cooked Kelp Fronds',
+				uncookable: true,
+				isveggie: true,
+				veggie: 1,
+				health: 0,
+				hunger: calories_tiny,
+				sanity: -sanity_tiny,
+				perish: perish_med,
+				stack: stack_size_smallitem,
+				mode: 'together'
+			},
+	
+	kelp_dried: {
+				name: 'Dried Kelp Fronds',
+				uncookable: true,
+				isveggie: true,
+				veggie: 1,
+				health: healing_tiny,
+				hunger: calories_tiny,
+				sanity: sanity_small,
+				perish: perish_preserved,
+				stack: stack_size_smallitem,
+				mode: 'together'
+			},
+
+	moonbutterflywings: {
+				name: 'Moon Moth Wings',
+				isveggie: true,
+				decoration: 2,
+				health: healing_medsmall,
+				hunger: calories_tiny,
+				sanity: sanity_med,
+				perish: perish_fast,
+				stack: stack_size_smallitem
+			},
+	
+	moon_tree_blossom: {
+				name: 'Moon Moth Wings',
```

Comment:
You forgot to change the name, dunno about other things.

Reply:
Thank you for that, I corrected it.